src/wolframite/core.clj

```diff
@@ -38,9 +38,10 @@
    [wolframite.base.parse :as parse]
    [wolframite.impl.jlink-instance :as jlink-instance]
    [wolframite.impl.protocols :as proto]
-   [wolframite.runtime.jlink :as jlink]
-   [wolframite.runtime.system :as system]
-   [wolframite.runtime.defaults :as defaults]
+   [wolframite.runtime
+    [jlink :as jlink]
+    [system :as system]
+    [defaults :as defaults]]
```

Comment:
I actually prefer the simpler, original approach, even though more verbose. YMMV :)

Reply:
Fair dos.